config.js

```diff
@@ -25,15 +25,17 @@ if (fs.existsSync(jsonPath)) {
 		host: env('HOST', jsonConfig.host),
 		port: Number(env('PORT', jsonConfig.port)),
 		cron: env('CRON', jsonConfig.cron),
-		chromeLaunchConfig: jsonConfig.chromeLaunchConfig || {}
+		chromeLaunchConfig: jsonConfig.chromeLaunchConfig || {},
+		numWorkers: Number(env('NUM_WORKERS', '2'))
```

Comment:
This should be `jsonConfig.numWorkers || 2`
```suggestion
		numWorkers: jsonConfig.numWorkers || 2
```

Reply:
Sure, I will.